CITATION.cff

```diff
@@ -0,0 +1,21 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "van Steen"
+  given-names: "Floyd"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+- family-names: "Mirketou"
+  given-names: "Anastasia"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+- family-names: "Acquaviva"
+given-names: "Marisa"
+- family-names: "Gonzalez"
```

Comment:
I think we should include all the members of the team as authors

Reply:
They are included